Source/Core/DolphinQt2/GameList/GameList.h

```diff
@@ -12,6 +12,7 @@
 
 #include "DolphinQt2/GameList/GameFile.h"
 #include "DolphinQt2/GameList/GameListModel.h"
+#include "DolphinQt2/GameList/TableDelegate.h"
```

Comment:
Done. I should make a separate PR to make lots of other things forwardly-declared, since I haven't been.